tests/unit/test_charm.py

```diff
@@ -1,12 +1,81 @@
 # Copyright 2022 Canonical Ltd.
 # See LICENSE file for licensing details.
 
+# pylint: disable=duplicate-code,protected-access
+
 """Spring Boot charm unit tests."""
+import ops.charm
+import pytest
 
+import exceptions
 
-import pytest
+
+def test_sprint_boot_pebble_layer(harness, patch):
```

Comment:
What if /app doesn't exist?

Reply:
The charm will enter blocking status with status message `Unknown Java application type`